Classes/Api/Orcid/OrcidClient.php

```diff
@@ -25,7 +25,7 @@
  * @subpackage dlf
  * @access public
  **/
-class Client
+class OrcidClient
```

Comment:
It's more consistent to keep the naming scheme `Classes/Api/Orcid/Client` and `Classes/Api/Viaf/Client` respectively. To avoid namespace duplicates when using both classes within the same file, declaration should be like:
`use Kitodo\Dlf\Api\Orcid\Client as OrcidClient;`